quickstart.sh

```diff
@@ -272,12 +234,12 @@ main() {
     printf '%s\n\n' "${color_good}Latest release of ${artifact_group}:${artifact_id} seems to be ${artifact_version}${color_reset}"
 
     printf '%s\n' "${color_title}Downloading $artifact_group:$artifact_id:$artifact_version:$artifact_classifier to $filename...${color_reset}"
-    artifact_group_with_slashes="${artifact_group//.//}"
-    artifact_url="https://dl.bintray.com/openzipkin/maven/${artifact_group_with_slashes}/${artifact_id}/$artifact_version/${artifact_id}-${artifact_version}${artifact_classifier_suffix}.jar"
+    artifact_url="${repo}/${artifact_group_with_slashes}/${artifact_id}/$artifact_version/${artifact_id}-${artifact_version}${artifact_classifier_suffix}.jar"
     fetch "$artifact_url" "$filename"
     verify_checksum "$artifact_url" "$filename"
-    verify_signature "$artifact_url" "$filename"
-    verify_signature "$artifact_url.md5" "$filename.md5"
+    # Disabled as it is assuming the GPG key is from Bintray, which it won't be. This is better than
+    # scaring people. In the future, we can do fancy things to determine which of the ASF keys it is.
```

Comment:
Or even verify only that it's one of the keys in the relevant KEYS file that was used to sign. We don't even necessarily care which one.

Reply:
yep this will work, needs a little logic as still we are mixed mode (ex is it bintray or apache as zipkin-aws for example, is still bintray signed!)